src/plugins/taxes-rates/resolvers/Mutation/deleteTaxRate.js

```diff
@@ -0,0 +1,35 @@
+import { decodeShopOpaqueId, decodeTaxRateOpaqueId } from "../../xforms/id.js";
```

Comment:
same questions as above

Reply:
@willopez same answer as the one I posted above